knotx-core/src/test/java/io/knotx/fragments/FragmentContentExtractorTest.java

```diff
@@ -37,8 +42,22 @@
   public void getUnwrappedContent_withFragment_expectDefinedContent(Fragment fragment,
       String expectedContentFileName) throws Exception {
 
-    final String expectedContent = FileReader.readText(expectedContentFileName);
+    final String expectedContent = readText(expectedContentFileName);
     final String unwrappedContent = FragmentContentExtractor.getUnwrappedContent(fragment);
     assertThat(unwrappedContent, equalToIgnoringWhiteSpace(expectedContent));
   }
+
+  @Coercion
+  public Fragment provideFragment(String fragmentContentFile) throws IOException {
```

Comment:
The problem here is connected with circular dependency between knotx-core and  knotx-junit. Can we fix this issue in new PR?